patches/minecraft/net/minecraft/world/item/DispensibleContainerItem.java.patch

```diff
@@ -0,0 +1,11 @@
+--- a/net/minecraft/world/item/DispensibleContainerItem.java
++++ b/net/minecraft/world/item/DispensibleContainerItem.java
+@@ -10,5 +_,8 @@
+    default void m_142131_(@Nullable Player p_150817_, Level p_150818_, ItemStack p_150819_, BlockPos p_150820_) {
+    }
+ 
++   @Deprecated //Forge: use the ItemStack sensitive version
+    boolean m_142073_(@Nullable Player p_150821_, Level p_150822_, BlockPos p_150823_, @Nullable BlockHitResult p_150824_);
++
++   boolean emptyContents(@Nullable Player p_150821_, Level p_150822_, BlockPos p_150823_, @Nullable BlockHitResult p_150824_, ItemStack itemStack);
```

Comment:
Can we name the stack parameter something useful like `container`?

Reply:
Good idea, done :)